@here/harp-mapview/lib/MapView.ts

```diff
@@ -103,6 +103,10 @@ export type MapAnchor<T extends THREE.Object3D = THREE.Object3D> = T & {
      * The position of this [[MapObject]] in [[GeoCoordinates]].
      */
     geoPosition?: GeoCoordinates;
+    /**
+     * The position of this [[MapObject]] in [[WorldCoordinates]]
```

Comment:
`WorldCoordinates` is not a type.

Reply:
Yeah... it was before "typedef" for Vec3, of course fixed that.